### ISSUE TITLE
Implement proxy

### ISSUE THREAD

**Issue by h3poteto** [h3poteto/whalebird-desktop#4794]
Refs: https://github.com/h3poteto/whalebird-desktop/issues/4653#issuecomment-1836192119

**Comment by h3poteto** [comment]
Ah, that's true. You can't open the settings screen before logging in, so you can't set up a proxy. Can you please open a new issue?

**Comment by dhs-rec** [comment]
Sure. See #5261.
